ui/desktop/app/routes/scopes/scope/projects.js

```diff
@@ -49,4 +53,24 @@ export default class ScopesScopeProjectsRoute extends Route {
     );
     return projects;
   }
+
+  @runEvery(2000)
+  async poller() {
+    const sessions =
+      await this.clientAgentSessions.getNewSessionsWithCredentials();
+
+    sessions.forEach((session) => {
+      new window.Notification(
+        this.intl.t('notifications.connected-to-target.title', {
+          target: session.alias,
+        }),
+        {
+          body: this.intl.t('notifications.connected-to-target.description'),
+        },
+      ).onclick = () => {
+        window.location.href = `serve://boundary/#/scopes/global/projects/sessions/${session.session_authorization.session_id}`;
```

Comment:
I wasn't sure if hardcoding the scope to `global` would be an issue but it turns out as long as we can list the scopes, it should be fine. It seems like it still works even if we don't have access to the global scope (and get a 403) but it'll still lead there correctly. 

I'm not sure where else to go, I could just use the default scope of wherever they're currently in but an alias could lead to a different scope.

After typing all this out, I realize I could also look through the resolvable aliases, find the target, find the scope that target is in and use that, which might be better. I'll try that